src/components/sidenav/sidenav.ts

```diff
@@ -325,6 +325,7 @@ export class MdSidenavLayout implements AfterContentInit {
     }
   }
 
+  /* tslint:disable:no-unused-variable */
```

Comment:
This has the same effect without wrapping the entire block if you prefer.
![screen shot 2016-03-08 at 1 23 41 pm](https://cloud.githubusercontent.com/assets/8420490/13613288/1c07751e-e531-11e5-9f32-9665010849f9.png)